lib/plugins/rollback/tests/rollback.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Rollback = require('../index');
+const Serverless = require('../../../Serverless');
+
+describe('Rollback', () => {
+  let rollback;
+  let serverless;
+
+  beforeEach(() => {
+    serverless = new Serverless();
+    rollback = new Rollback(serverless);
+  });
+
+  describe('#constructor()', () => {
+    it('should have the command rollback', () => {
+      // eslint-disable-next-line no-unused-expressions
```

Comment:
no-unused-expressions is a useful rule to identify unused expressions which have no effect on the state of the program. Unfortunately chai is written in a way that it actually leverages such unused expression and in the case of chai the rule is broken.

This is valid chai: `expect(true).to.be.true;`

What I did in all my projects using chai is to use this chai extension: https://www.npmjs.com/package/dirty-chai

In this case you can do
```
expect(true).to.be.true; // default chai
expect(true).to.be.true(); // dirty chai - no-unused-expressions is not erroring
```

An alternative would be to do `.to.be.equal(true)`. We actually use this all over, but I'm not sure if this actually checks for true or just truthy. It's on my list to investigate. Should I change it to this one for now since we use it all over.